StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/MaintainabilityRules/SA1413CodeFixProvider.cs

```diff
@@ -115,5 +121,11 @@ private static SyntaxNode RewriteAnonymousObjectInitializer(AnonymousObjectCreat
             var fixedInitializer = initializer.WithInitializers(newInitializerExpressions);
             return fixedInitializer;
         }
+
+        private static TextChange GetEnumMemberTextChange(Diagnostic diagnostic, SyntaxNode syntaxRoot)
+        {
+            var member = (EnumMemberDeclarationSyntax)syntaxRoot.FindNode(diagnostic.Location.SourceSpan);
+            return new TextChange(diagnostic.Location.SourceSpan, member.ToString() + ",");
+        }
```

Comment:
I changed the code to only use my method and it still passes all the unit tests. Not sure how to test if it performs better but it does a lot less work: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/pull/2245/commits/f983b5f165190e0083a78a866d3cd908196f6b2f#diff-9ac99be9ffc83f0bfb6aa1415e965bccR49.